src/containers/BooksForm.js

```diff
@@ -7,10 +11,33 @@ const BooksForm = () => {
     </option>
   ));
 
+  const [book, setBook] = useState({ title: '', category: '' });
+  const dispatch = useDispatch();
+
+  let title = '';
+  let category = '';
+
+  const handleChange = (e) => {
+    if (e.target.name === 'title') {
+      title = e.target.value;
+      setBook((state) => ({ ...state, title }));
+    }
+    if (e.target.name === 'category') {
+      category = e.target.value;
+      setBook((state) => ({ ...state, category }));
+    }
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    dispatch(createAction({ ...book, id: Math.floor(Math.random() * 100) }));
```

Comment:
- [ ] [REQUIRED] you should `reset the STATE `once the book is created.  [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/HandleSubmit.png)